projects/fsastorefrontlib/src/occ/services/default-occ-user-config.ts

```diff
@@ -4,7 +4,7 @@ export const occUserConfig: FSOccConfig = {
   backend: {
     occ: {
       endpoints: {
-        user: 'fsusers/${userId}',
```

Comment:
I think that fsUsers will be needed for some requests like registration because we need to send date of birth. Probably with this change, it's not saved on customer, please check that

Reply:
Same for updateUserRequest. i think that we need to introduce fsUser endpoint which will be 'fsUsers/{userId} and than override those 2 methods in occ-user-adapter to use our endpoints.